From fcf13f967357db93659cb471ac401d35049b6fd2 Mon Sep 17 00:00:00 2001
From: Rene Engelhard <rene@debian.org>
Date: Tue, 4 Jul 2023 20:36:46 +0200
Subject: [PATCH] don't use SSE in skia on "INTEL"

as i386/i686 in Debian is non-MMX, non-SSE, cf.
https://wiki.debian.org/ArchitectureSpecificsMemo#i386-1

Regression from 9c9a711ac5d8f32ac318d0e4ecab7b3a26bc2150 which claims
to have set the define instead of skia_sk_cpu_sse_level_0_by_default.patch.1
but actually sets it to SSE2...

Change-Id: I4157a580470732fd7bbc5cf1b2384b76182340cd
---
 external/skia/Library_skia.mk | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/external/skia/Library_skia.mk b/external/skia/Library_skia.mk
index 4b7d9ee56407..044811a2110e 100644
--- a/external/skia/Library_skia.mk
+++ b/external/skia/Library_skia.mk
@@ -20,7 +20,8 @@ $(eval $(call gb_Library_add_defs,skia,\
     -DSKIA_IMPLEMENTATION=1 \
     -DSKIA_DLL \
     -DSK_USER_CONFIG_HEADER="<$(BUILDDIR)/config_host/config_skia.h>" \
-    $(if $(filter INTEL X86_64,$(CPUNAME)),-DSK_CPU_SSE_LEVEL=SK_CPU_SSE_LEVEL_SSE2) \
+    $(if $(filter INTEL,$(CPUNAME)),-DSK_CPU_SSE_LEVEL=0) \
+    $(if $(filter X86_64,$(CPUNAME)),-DSK_CPU_SSE_LEVEL=SK_CPU_SSE_LEVEL_SSE2) \
 ))
 
 # SK_DEBUG controls runtime checks and is controlled by config_skia.h and depends on DBG_UTIL.
-- 
2.39.2

